eox_tenant/apps.py

```diff
@@ -51,3 +51,6 @@ def ready(self):
         """
         from eox_tenant.permissions import load_permissions
         load_permissions()
+
+        from eox_tenant.monkey_patch import load_monkey_patchs_overrides
+        load_monkey_patchs_overrides()
```

Comment:
Either `patches` or `patch`. I think monkey_patch_overrides is the correct version